src/DataStructures/SliceVariables.hpp

```diff
@@ -96,11 +96,11 @@ void add_slice_to_data(
   const size_t volume_grid_points = extents.product();
   const size_t slice_grid_points = extents.slice_away(sliced_dim).product();
   ASSERT(volume_vars->number_of_grid_points() == volume_grid_points,
-         "volume_vars has wrong number of grid points.  Expected "
+         "volume_vars has wrong number of grid points. Expected "
```

Comment:
Could you change this error message in a separate commit? It's unrelated to converting error tests

Reply:
I just dropped the change